README.md

```diff
@@ -53,7 +53,7 @@ The charm is by default using a layer 2 configuration, and the ip range
 or use the juju config command line to edit it post-deployment. 
 
     juju add-model metallb-system
-    juju deploy cs:~containers/metallb-bundle
+    juju deploy cs:~containers/metallb
```

Comment:
Will it be metallb or metallb-operator ?

Reply:
Since it's the bundle, comprising more than one operator charm, I was just assuming it would be `metallb`.  Open to other options, though.